tembo-stacks/src/apps/http.yaml

```diff
@@ -1,6 +1,6 @@
 name: !http
 appServices:
-  - image: postgrest/postgrest:v10.0.0
+  - image: postgrest/postgrest:v12.2.5
```

Comment:
any breaking changes from v10 -> v12? The things on my radar would be these env vars: `PGRST_DB_URI,  PGRST_DB_SCHEMA, PGRST_DB_ANON_ROLE`

Reply:
Dunno, will try stuff in dev once merged.